tabular/src/autogluon/tabular/models/text_prediction/text_prediction_v1_model.py

```diff
@@ -0,0 +1,280 @@
+                                                feature_columns=list(X_train.columns),
+                                                label_columns=[self._label_column_name],
+                                                problem_types=[problem_type],
+                                                label_shapes=[label_shape],
+                                                stopping_metric=stopping_metric,
+                                                log_metrics=log_metrics,
+                                                output_directory=os.path.join(self.path, self.name),
+                                                logger=logger,
+                                                base_config=None,
+                                                search_space=search_space)
+        return column_properties
+
+    def _get_default_auxiliary_params(self) -> dict:
+        default_auxiliary_params = super()._get_default_auxiliary_params()
+        extra_auxiliary_params = dict(
+            get_features_kwargs=dict(
+                valid_raw_types=[R_INT, R_FLOAT, R_OBJECT],
+                invalid_special_types=[S_TEXT_NGRAM, S_TEXT_AS_CATEGORY, S_TEXT_SPECIAL],
+            ),
+        )
+        default_auxiliary_params.update(extra_auxiliary_params)
+        return default_auxiliary_params
+
+    @classmethod
+    def _get_default_ag_args(cls) -> dict:
+        default_ag_args = super()._get_default_ag_args()
+        extra_ag_args = {'valid_stacker': False}
+        default_ag_args.update(extra_ag_args)
+        return default_ag_args
+
+    def _set_default_params(self):
+        try:
+            from autogluon.text.text_prediction.dataset import TabularDataset
+            from autogluon.text.text_prediction.text_prediction import ag_text_prediction_params
+        except ImportError:
+            raise ImportError(AG_TEXT_IMPORT_ERROR)
+        super()._set_default_params()
+        self.params = ag_text_prediction_params.create('default_no_hpo')
+
+    def _fit(self, X_train: pd.DataFrame, y_train: pd.Series,
+             X_val: Optional[pd.DataFrame] = None,
+             y_val: Optional[pd.Series] = None,
+             time_limit: Optional[int] = None, **kwargs):
+        """The internal fit function
+
+        Parameters
+        ----------
+        X_train
+            Features of the training dataset
+        y_train
+            Labels of the training dataset
+        X_val
+            Features of the validation dataset
+        y_val
+            Labels of the validation dataset
+        time_limit
+            The time limits for the fit function
+        kwargs
+            Other keyword arguments
+
+        """
+        try:
+            import mxnet as mx
+            from autogluon.text.text_prediction.dataset import TabularDataset
+            from autogluon.text.text_prediction.text_prediction import get_recommended_resource
+        except ImportError:
+            raise ImportError(AG_TEXT_IMPORT_ERROR)
+
```

Comment:
Add TODO: Raise exception if no GPU (With added functionality to override and use CPU anyway as an AG_args_fit parameter)

Reply:
In fact, I just noticed that it may also be reasonable for users to try out BERT on CPU (not completely disable it). May be we can just raise a warning.